rest_api/api-spec.yaml

```diff
@@ -696,6 +696,8 @@ definitions:
       A simple enum and with optional value, used to constrain and define the
       behavior of Assets or Offers
     type: object
+    required:
+      - type
```

Comment:
As per our recent Slack discussion, `target` and `targetQuantity` should be removed from the `required` field of `NewOfferBody`